std/haxe/io/Bytes.hx

```diff
@@ -130,6 +142,9 @@ class Bytes {
 		#end
 	}
 
+	/**
+	  Returns a new `Bytes` instance that contains a copy of this instance's `len` bytes, starting at index `pos`. 
```

Comment:
"… that contains a copy of `len` bytes of `this` instance, starting at index `pos`."